src/app/loadout-drawer/selectors.ts

```diff
@@ -15,6 +19,66 @@ export const loadoutsSelector = createSelector(
       : emptyArray<Loadout>()
 );
 
+export interface LoadoutsByItem {
```

Comment:
I wouldn't expect this to have the `LoadoutItem` in this going off the name but I don't have a better suggestion that `LoadoutsAndItemsByItemId` which is a bit long winded.